### PR TITLE
FE-415-Flow3

### PR DESCRIPTION
**Pull request by datnguyenner**
https://snack.expo.dev/@seven.me/github.com-sevenmeinc-vision@fe-415-flow3

### CODE REVIEW

File: src/navigations/Flow3Navigator/index.js [hunk 1]
```diff
@@ -1,18 +1,32 @@
...
 import PromptVideo from '../../screens/SucHab01Screens/PromptVideo'
 import PromptText from '../../screens/SucHab01Screens/PromptText'
+import PostActivity from '../../screens/SucHab01Screens/PostActivity'
+import BreatheScreen from '../../screens/SucHab01Screens/Breathe'
+import PreviewScreen from '../../screens/SucHab01Screens/PreviewScreen'
+import VideoPlayer from '../../screens/PositiveThinking/video'
 import { content } from '../../../assets/content'
 
 const Stack = createStackNavigator()
+const { prompts, postActivity } = content.successfulHabits
```
Comment (JayRVigilla) [inline src/navigations/Flow3Navigator/index.js:19]: You are bringing in content for Successful Habits. You have to hard code your prompts or create an additional content object. The prompts going to TextPrompt and VideoPrompt are for a different exercise

File: src/navigations/Flow3Navigator/index.js [hunk 2]
```diff
@@ -28,44 +42,272 @@ const Flow3Navigator = () => {
...
+              current={2}
+              total={5}
+              navigation={navigation}
+            />
+          )
+        })}
+        children={(props) => (
+          <PromptText
+            {...props}
+            prompt={prompts[0].prompt}
```
Comment (JayRVigilla) [inline src/navigations/Flow3Navigator/index.js:225]: prompt from successful habits

File: src/navigations/Flow3Navigator/index.js [hunk 3]
```diff
@@ -28,44 +42,272 @@ const Flow3Navigator = () => {
...
+              current={4}
+              total={5}
+              navigation={navigation}
+            />
+          )
+        })}
+        children={(props) => (
+          <PromptText
+            {...props}
+            prompt={prompts[1].prompt}
```
Comment (JayRVigilla) [inline src/navigations/Flow3Navigator/index.js:247]: prompt from successful habits

File: src/navigations/Flow3Navigator/index.js [hunk 4]
```diff
@@ -28,44 +42,272 @@ const Flow3Navigator = () => {
...
+              current={3}
+              total={5}
+              navigation={navigation}
+            />
+          )
+        })}
+        children={(props) => (
+          <BreatheScreen
+            {...props}
+            prompt={prompts[1]}
```
Comment (JayRVigilla) [inline src/navigations/Flow3Navigator/index.js:269]: prompt from successful habits, though not harmful due to default in `BreatheScreen`

File: src/navigations/Flow3Navigator/index.js [hunk 5]
```diff
@@ -28,44 +42,272 @@ const Flow3Navigator = () => {
...
             />
           ),
-          presentation: 'card'
+          headerShown: isPreview
         })}
+        children={(props) => (
+          <PromptVideo
+            {...props}
+            setIsPreview={setIsPreview}
+            prompt={prompts[0].prompt}
```
Comment (JayRVigilla) [inline src/navigations/Flow3Navigator/index.js:129]: prompt from successful habits

File: src/navigations/Flow3Navigator/index.js (hunk 1 above)
Comment (datnguyenner) [reply src/navigations/Flow3Navigator/index.js:19]: I hardcoded the prompts that were different from successful habit but I can create separate content for it

File: src/navigations/Flow3Navigator/index.js [hunk 6]
```diff
@@ -28,44 +42,272 @@ const Flow3Navigator = () => {
...
+        options={({ navigation }) => ({
+          header: () => (
+            <BenefitProgressHeader
+              title={'Benefits List'}
+              navigation={navigation}
+            />
+          )
+        })}
+        children={(props) => (
+          <PostActivity {...props} postActivity={postActivity} />
```
[the remark this answers is not in the document]
Comment (JayRVigilla) [reply src/navigations/Flow3Navigator/index.js:309]: Are there different PostActivity prompts in different Figmas? I was looking at the prompt here:
https://www.figma.com/file/JIEYtn6ofei0Qge3vyqE5c/Immersive-Experiences%3A-Design-Exploration?node-id=1394%3A10602

**Pull request merged**